sycl/doc/extensions/experimental/sycl_ext_oneapi_kernel_compiler.asciidoc

```diff
@@ -292,6 +310,10 @@ _Returns:_ The newly created kernel bundle, which has `executable` state.
 
 _Throws:_
 
+* An `exception` with the `errc::invalid` error code if any of the devices in
+  `devs` return `false` for `ext_oneapi_can_build` with the source language
+  `lang`.
+
```

Comment:
This seems right, but this existing exception is now redundant:

> An `exception` with the `errc::invalid` error code if any of the devices in `devs` does not support compilation of kernels in the source language of `sourceBundle`.

Kind of a nit .. but it's probably better to say:

> ... with the source language of `sourceBundle`.

rather than:

> ... with the source language `lang`.

because there is no parameter to this function named `lang`.